hack/test/e2e-firecracker.sh

```diff
@@ -7,6 +7,18 @@ source ./hack/test/e2e.sh
 PROVISIONER=firecracker
 CLUSTER_NAME=e2e-${PROVISIONER}
 
+case "${REGISTRY:-false}" in
+  registry.ci.svc:5000)
+    REGISTRY_ADDR=`python -c "import socket; print socket.gethostbyname('registry.ci.svc')"`
```

Comment:
resolve hostname before passing it to the firecracker; firecracker runs with default DNS servers (8.8.8.8), so it can't resolve cluster-local addresses